master/internal/db/postgres_tasks.go

```diff
@@ -247,6 +248,73 @@ func (db *PgDB) UpdateAllocationProxyAddress(a model.Allocation) error {
 	return err
 }
 
+// Accelerators object of string slice type.
+type Accelerators []string
+
+// Scan database row of json type into Accelerators object.
+func (a *Accelerators) Scan(src any) error {
+	bytes, ok := src.([]byte)
+	if !ok {
+		return errors.New("src value cannot cast to []byte")
+	}
+	err := json.Unmarshal(bytes, &a)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// GetTaskAcceleratorData gets the accelerator data for the given task.
+func (db *PgDB) GetTaskAcceleratorData(id string) ([]*apiv1.AcceleratorData, error) {
+	rows, err := db.sql.Query(
```

Comment:
Changed GetTaskAccelerationData query to Bun but for AddAllocationAcceleratorData I think it is simpler with Query because we'd have to create a new struct to pass into Bun().NewInsert because the table name(allocation_accelerators) differs from the existing struct(AcceleratorData)